### PR TITLE
Added kubestack to tool list

### PR DESCRIPTION
**Pull request by Spazzy757**
# Changelog

Added KubeStack to the list of tools. KubeStack, in essence, is a framework that sets up production-grade Kubernetes on Azure, Amazon or Google. It comes with some example pipeline files to setup on Jenkins, Gitlab, GitHub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Modern software development practices _assume_ support for reviewing changes, tr
 - [Jenkins X](https://jenkins-x.io/) - a CI/CD platform for Kubernetes that provides pipeline automation, built-in GitOps and preview environments
 - [Weave Cloud](https://www.weave.works/product/cloud/) - GitOps experience, workflows and dashboard for your cluster(s) (commercial product from Weaveworks)
 - [Werf](https://werf.io) - GitOps tool with advanced features to build images and deploy them to Kubernetes (integrates with any existing CI system)
+- [KubeStack](https://www.kubestack.com/) - GitOps framework using Terraform for Cloud Kubernetes distros (AKS, GKE and EKS) with CI/CD examples for common tools.
```
Comment (dimitropoulos) [inline README.md:58]:
```suggestion
- [KubeStack](https://www.kubestack.com/) - GitOps framework using Terraform for Cloud Kubernetes distros (AKS, GKE and EKS) with CI/CD examples for common tools
```

Comment (dimitropoulos) [inline README.md:58]:
```suggestion
- [KubeStack](https://www.kubestack.com/) - GitOps framework using Terraform for Cloud Kubernetes distros (AKS, GKE, and EKS) with CI/CD examples for common tools.
```

Comment (Spazzy757) [reply README.md]: Thanks for the update

**Comment by dimitropoulos** [comment]
looks like something went wrong, the period was removed but the oxford comma was also removed.  I made the commit suggestion in two separate segments for the same line, which I can now see I shouldn't have done because it confused GitHub.  Mind just fixing that up? otherwise we're good to go!

**Comment by Spazzy757** [comment]
Ofcourse I will do it now

**Comment by Spazzy757** [comment]
I also rebased to a single commit hope thats okay

**Pull request merged**